.github/workflows/ci.yml

```
@@ -10,4 +10,4 @@ jobs:
     uses: ljharb/actions/.github/workflows/node-majors.yml@main
     with:
       range: '>= 14.17'
-      command: 'npm run tests-only && npm run licenses'
+      command: 'npm run engines && npm run tests-only && npm run licenses'
```

we don't need this to run in every node version - so instead, i'd add a `postlint` script of `ls-engines` (`npm run engines` isn't a thing)
```suggestion
      command: 'npm run tests-only && npm run licenses'
```